client/js/components/procedure/basicSettings/DpBasicSettings.vue

```diff
@@ -156,6 +189,34 @@ export default {
         })
     },
 
+    handleAddonRequest () {
+      const payload = this.createAddonPayload()
+
+      const addonRequest = dpApi({
+        headers: {
+          ...(dplan.csrfToken && { 'x-csrf-token': dplan.csrfToken }) // TODO: should be adjusted in UI: api2defaultHeaders
```

Comment:
Why isn't it adjusted in demosplan-ui? It's a very small change, isn't it?

Reply:
yes, i was sick for two weeks, so was not able to do it :) i will do it

Reply:
Ah, of course. :) (Hope you're better!)

Reply:
I need to look that up, as far as I know the header should already be a part of `dpApi` and the backend should check for either the `x-csrf-token ` header or use the hidden input in case no such header is provided

Reply:
Long story short, we do not need it here and for occurrences we need it, it is already included in dpApi.

Reply:
hmm, we have `apiDefaultHeaders` and `api2DefaultHeaders`. We set it explicitly in the headers for API 1.0 but not for API 2.0. I wasn't sure why, so I thought it might be a fix to add it there

Reply:
We actually only need it for form posts and I have added the csrf token to dpApi a while ago so we do not need to include the hidden csrf input field in all forms (just for convenience reasons)  https://github.com/demos-europe/demosplan-core/pull/3999/commits/2eca50b44d393164e8b8d6d9aba0ef62477b9dcb

Reply:
hm, but if you remove it, the lilla messages will reappear on the POST request. Or you want to add in demosplan ui?